src/smartbanner.js

```diff
@@ -189,6 +189,12 @@ export default class SmartBanner {
     return this.options.hidePath ? this.options.hidePath : '/';
   }
 
+  get parentElement() {
+    const parentElement = this.options.parentElement ? document.querySelector(this.options.parentElement) : null;
+
+    return parentElement || document.querySelector('body');
```

Comment:
Let's be even more readable here instead of the ternary:

```suggestion
    if (this.options.parentElement) {
      return document.querySelector(this.options.parentElement);
    }
    return document.querySelector('body');
```